spring-cloud-dataflow-docs/pom.xml

```diff
@@ -92,21 +101,11 @@
 												<include name="**/*.html" />
 											</fileset>
 										</copy>
-										<copy todir="${basedir}/target/contents/reference/htmlsingle">
-											<fileset dir="${basedir}/src/main/docbook">
-												<include name="**/*.css" />
-												<include name="**/*.png" />
-												<include name="**/*.gif" />
-												<include name="**/*.jpg" />
-											</fileset>
+										<copy todir="${basedir}/target/contents/reference/htmlsingle/images">
+											<fileset dir="${basedir}/target/generated-docs/images"/>
```

Comment:
Yes, that would seem to be desirable since we may eventually drop docbook as it is just a gateway to creating pdf, which we never link directly to anymore.